docs/docs-content/clusters/edge/networking/vxlan-overlay.md

```diff
@@ -32,6 +32,11 @@ The Analytics team of a manufacturing company is deploying an Edge host to their
 |---------------------|-----------------------|
 | Upon recovery, each Kubernetes component inside in the Edge host requests an IP address from the DHCP server, and receives a different IP address than their original IP address before the outage happened. Since Kubernetes expects several components in the control plane to have stable IP addresses, the cluster becomes non-operational and assembly line is unable to resume operations | Each Kubernetes component inside in the Edge host has a virtual IP address in the overlay network. Upon recovery, their IP addresses in the overlay network remain the same despite their IP addresses changing in the underlying DHCP network. The Edge host is able to assume its workload and the assembly line resumes operations | 
 
+## Limitations
+- When adding multiple nodes to an existing cluster with overlay enabled, failure to add one node will block the addition of the other nodes.
+
+- When deleting an Edge host from a cluster with overlay enabled, ensure the node doesn't have the `palette-webhook` pod on it, or the node will be stuck in the deleting state. You can use the command `kubectl get pods -all-namespaces -output wide` to identify which node `palette-webhook` is on.
```

Comment:
Nianyu says they would need to manually move the pod because "there is toleration on that pod and draining the node will not help". He agreed that this would be a complicated thing to do and it's best we don't go into it here.